### PR TITLE
Make registration attendance system public

### PR DESCRIPTION
**Pull request by asgrim**
Stuff.

### CODE REVIEW

File: public-src/phphants/scss/style.scss [hunk 1]
```diff
@@ -182,6 +182,10 @@ header .nav-button:active{
     opacity: .5;
 }
 
+header .login-status-text {
+    margin-top: -10px; font-size: 14px;
```
Comment (davidyell) [inline public-src/phphants/scss/style.scss]: Other Sass doesn't seem to be on the same line. Could be a consistency issue?

**Comment by doctor-data** [comment]
Other than @davidyell comment, all looks good

**Comment by asgrim** [comment]
Well, @doctor-data approved, so merging.

File: templates/layout/header.phtml [hunk 2]
```diff
@@ -26,6 +26,14 @@ declare(strict_types=1);
                 <a href="<?= $this->escapeHtmlAttr($this->url('contact')); ?>">Contact</a>
             </li>
         </ul>
+        <div class="pull-right login-status-text">
+            <?php if ($this->user()->isLoggedIn()): ?>
+                Logged in as <?= $this->user()->get()->displayName(); ?>. View your <a href="<?= $this->escapeHtmlAttr($this->url('account-dashboard')); ?>">Dashboard</a>.
```
Comment (lboynton) [inline templates/layout/header.phtml]: Escaping on `$this->user()->get()->displayName();` ?

File: templates/app/account/login.phtml [hunk 3]
```diff
@@ -42,4 +42,8 @@ $this->headTitle($title);
         </li>
     </ul>
 
+    <h1>If you don't have an account...</h1>
+
+    <p>You can <a href="<?= $this->url('account-register'); ?>">register now</a> with your email.</p>
```
Comment (lboynton) [inline templates/app/account/login.phtml]: Need to `escapeHtmlAttr()` on the URL

**Pull request merged**